### PR TITLE
[OSDOCS-7641]: SRE cluster access

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.13+
Issue:

https://issues.redhat.com/browse/OSDOCS-7641
Link to docs preview:

https://64354--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/sre-cluster-access.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa_policy_service_definition/rosa-sre-access.adoc
+
+:_content-type: CONCEPT
+[id="sre-cluster-access_{context}"]
+= SRE cluster access
+
+SRE access to {product-title} (ROSA) clusters can get full cluster administrative access through an escalation procedure; this procedure consists of several layers of authentication and multiple streams of approval, all of which are managed by strict company policy. All authentication attempts to access a cluster, as well as changes made within a ROSA cluster, are recorded within audit logs, along with the specific account identity of the SRE responsible for those actions. These audit logs help ensure that all changes made by SREs to a customer's ROSA cluster adhere to the strict policies and procedures that make up Red Hat's managed services guidelines.
```
Comment (EricPonvelle) [inline modules/sre-cluster-access.adoc:9]: Nit: I found this compound sentence a bit hard to read.

"SRE access to {product-title} (ROSA) clusters can get full cluster administrative access through an escalation procedure; this procedure consists of several layers of authentication and multiple streams of approval, all of which are managed by strict company policy." 

I think just changing the semicolon to a period would be sufficient, but you may want a second opinion. It just reads long.

File: modules/sre-cluster-access.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+** Once authorized, SRE logs into the fleet management plane and receives a service account token that the fleet management plane created. The token is valid for twelve minutes. Once the token is no longer valid, it is deleted.
+
+** With access granted to the fleet management plane, the SRE will use various methods to access clusters, depending on network configuration.
+
+*** Accessing a private network cluster: Request is sent through a specific Network Load Balancer (NLB) using an encrypted HTTP connection. The NLB contains an IP allow-list so the APIs will accept connections from a specific set of IPs of which the fleet management plane contains.
+
+*** Accessing a PrivateLink cluster: Request is sent to the Red Hat Transit Gateway, which then connects to a Red Hat VPC per region. The VPC that receives the request will be dependent on the target private cluster’s region. Within the VPC there is a private subnet which contains the PrivateLink endpoint to the customer’s PrivateLink cluster.
```
Comment (EricPonvelle) [inline modules/sre-cluster-access.adoc:23]: @jneczypor can chime in, but I think you should include a comma:

s/Within the VPC there is a private subnet/Within the VPC, there is a private subnet

**Review by EricPonvelle — COMMENTED** [review]
Very small language items but nothing too crazy.

File: modules/sre-cluster-access.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+:_content-type: CONCEPT
+[id="sre-cluster-access_{context}"]
+= SRE cluster access
+
+SRE access to {product-title}
+ifdef::openshift-rosa[]
+(ROSA)
+endif::openshift-rosa[]
+clusters can get full cluster administrative access through an escalation procedure; this procedure consists of several layers of authentication and multiple streams of approval, all of which are managed by strict company policy. All authentication attempts to access a cluster, as well as changes made within a cluster, are recorded within audit logs, along with the specific account identity of the SRE responsible for those actions. These audit logs help ensure that all changes made by SREs to a customer's cluster adhere to the strict policies and procedures that make up Red Hat's managed services guidelines.
```
Comment (kmcolli) [inline modules/sre-cluster-access.adoc:13]: I'm not sure I'd lead with full cluster admin access - escalated procedure - I'd move this to the end.  The escalated procedure is only required when the SRE needs to access non openshift-* namespaces.

File: modules/sre-cluster-access.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+** Once authorized, SRE logs into the fleet management plane and receives a service account token that the fleet management plane created. The token is valid for twelve minutes. Once the token is no longer valid, it is deleted.
+
+** With access granted to the fleet management plane, SRE uses various methods to access clusters, depending on network configuration.
+
+*** Accessing a private network cluster: Request is sent through a specific Network Load Balancer (NLB) using an encrypted HTTP connection. The NLB contains an IP allow-list so the APIs accept connections from a specific set of IPs of which the fleet management plane contains.
```
Comment (kmcolli) [inline modules/sre-cluster-access.adoc:26]: Accessing a private network cluster: Request is sent through a specific Network Load Balancer (NLB) using an encrypted HTTP connection on port 6443.

File: osd_architecture/osd_policy/osd-sre-access.adoc [hunk 5]
```diff
@@ -4,4 +4,8 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: osd-sre-access
 [id="osd-sre-access"]
 = SRE and service account access
-////
\ No newline at end of file
+
```
Comment (bmcelvee) [inline osd_architecture/osd_policy/osd-sre-access.adoc:7]: I think the first set of //// is causing the build failure

Comment (mletalie) [reply osd_architecture/osd_policy/osd-sre-access.adoc:7]: Hi @bmcelvee,  I think you are correct :( Just got off my 1:1 with Andrew, working on this now....

**Pull request merged**